Code/Engine/Foundation/ezEngine.py

```diff
@@ -0,0 +1,245 @@
+import lldb.formatters.Logger
```

Comment:
What is this file for ?

Reply:
This are the visualizers for the Foundation containers mentioned in the PR description. Similar to the Foundation.natvis file this allows a nice view of various Foundation containers when debugging with LLDB. Without these debugging on linux is very cumbersome.